scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc

```diff
@@ -35,6 +35,10 @@ include::modules/cnf-allocating-multiple-huge-page-sizes.adoc[leveloffset=+2]
 
 include::modules/cnf-cpu-infra-container.adoc[leveloffset=+2]
 
+.Additional resources
+
+* xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus[Managing device interrupt processing for guaranteed pod isolated CPUs]
```

Comment:
I would hyphenate 'device interrupt' in the xref.
So rewrite as 'Managing device-interrupt processing...'